include/language-factory.php

```diff
@@ -41,18 +41,18 @@ public static function get( $language_data ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param WP_Term[] $terms List of language terms, with the type as array keys.
-	 *                         `post` and `term` are mandatory keys.
+	 * @param WP_Term[] $terms List of language terms, with the language taxonomies language as array keys.
+	 *                         `language` and `term_language` are mandatory keys.
 	 * @return PLL_Language
 	 *
 	 * @phpstan-param array{post:WP_Term, term:WP_Term}&array<string, WP_Term> $terms
```

Comment:
```suggestion
	 * @phpstan-param array{language:WP_Term, term_language:WP_Term}&array<string, WP_Term> $terms
```

Reply:
Done in 89fa58a0d18c2fd1480a57bdfb7d3ab477932abb